compair/static/modules/course/course-assignments-partial.html

```diff
@@ -51,10 +51,14 @@ <h2>{{filter}} <span ng-if="results.length != assignments.length">({{results.len
 
         <div class="col-sm-4 text-right">
             <form class="form-inline searchCourse" role="search" ng-show="assignments.length">
-                <div class="form-group filter">
+                <div class="form-group filter" id="assignment-filter">
                     <label>Show: &nbsp;</label>
                     <select ng-model="filter" ng-options="f for f in filters"></select>
                 </div>
+                <div class="form-group filter" id="assignment-sort" ng-hide="canMangeAssignments">
+                    <label>Order by: &nbsp;</label>
+                    <select ng-model="sortOrder" ng-options="s for s in sortOptions"></select>
```

Comment:
I was having an issue getting the frontend tests to work when I had added that directive. Do you know how to change the select value when using that? Specifically, `element.all(by.css("#assignment-sort select")).sendKeys(sort);` was not working.

Reply:
you can try this
```
element(by.css("#assignment_sort_chosen .chosen-single")).click()
element(by.cssContainingText("#assignment_sort_chosen .chosen-results li", sort)).click()
```

chosen js doesn't use the underlining select element (so it can more easily control the presentation of the select + search)

Reply:
Also I'm not sure but using `element.all` might not work with sendKeys (since element.all is a list not a specific element)